src/screens/General/Dashboard.js

```diff
@@ -273,226 +264,242 @@ class Dashboard extends Component {
 
 	renderCommitteePanel() {
 		const {
-			touchCommittee,
-			textColor
+			committeeItemContainer,
+			textColor,
+			committeesPanelContainer,
+			committeesListContainer,
+			selectCommitteesIcon,
+			committeeNameContainer,
+			leaderboardArrow,
+			committeesPlaceHolder,
+			gold
 		} = styles;
+
 		const {
 			committeesList,
-			changeUserCommittees
+			changeUserCommittees,
+			userCommittees
 		} = this.props;
 
-		if (!this.props.userCommittees || !committeesList)
-			return (
-				<View style = {{ flexDirection: "row", flex: 1, height: "100%", backgroundColor: "#21252b", alignItems: "center" }}>
-					<View style = {{ height: "100%", justifyContent: "flex-start", alignItems: "center", flex: 0.25, paddingTop: "5%" }}>
-						<Ionicons
-							name = "ios-information-circle"
-							size = { dimension.height * 0.028 }
-							onPress = { () => Actions["CommitteesD"]({ screen: "dashboard" }) }
-							style = {{ color: "#FECB00" }}
-						/>
-					</View>
-					<View style = {{ flex: 1, backgroundColor: "#21252b", justifyContent: "space-evenly", height: "80%" }}>
-						<View style = {{}}>
-							<Text style = { [textColor, { fontSize: dimension.width * 0.03 }] }>Add your main committees!</Text>
-						</View>
-					</View>
+		let content = null;
+		let committeesArray = null;
+
+		if (!this.props.userCommittees || !committeesList) {
```

Comment:
This seems like a lot of code to either render some text or some text and an icon. It's hard to read on github but if possible try to refactor. If truly not possible then don't worry about it.

Reply:
Couldn't refactor at the moment, will look into it more after the update

Reply:
Please an issue about any change that you couldn't address